index.html

```diff
@@ -915,17 +915,55 @@ <h5 class="modal-title-craft"></h5>
                                     </table>
 
                                     <table class="parameter cf">
+                                        <thead>
+                                            <tr>
+                                                <th scope="row" colspan="6">Feedforward</th>
+                                            </tr>
+                                        </thead>
                                         <tbody>
                                             <tr>
-                                                <td name="ptermSRateWeight" class="bf-only">
-                                                    <label>P-Term
-                                                        <br>&nbsp;
+                                                <td name="feedforwardTransition" class="bf-only">
+                                                    <label>Transition
                                                         <br/>&nbsp;</label>
                                                     <input type="number" step="0.01" min="0" max="999" />
                                                 </td>
-                                                <td name="feedforward_transition" class="bf-only">
```

Comment:
Why the transition has been removed? Betaflight 4.2 uses it if I'm not wrong...

Reply:
I think `ptermSRateWeight` is removed, but transition is still there, and still shows up normally when looking at 4.2 logs.